src/main/webapp/app/entities/quiz-exercise/quiz-exercise.component.ts

```diff
@@ -32,6 +32,37 @@ export class QuizExerciseComponent implements OnInit, OnDestroy {
     reverse: boolean;
     courseId: number;
 
+    /**
+     * Exports given quiz questions into json file
+     * @param quizQuestions Quiz questions we want to export
+     * @param exportAll If true exports all questions, else exports only those whose export flag is true
+     */
+    static exportQuiz(quizQuestions: any, exportAll: boolean) {
+        const questions = [];
+        for (const question of quizQuestions) {
+            if (exportAll === true || question.exportQuiz === true) {
+                delete question.questionStatistic;
+                questions.push(question);
+            }
+        }
+        if (questions.length === 0) {
+            return;
+        }
+        const quizJson = JSON.stringify(questions);
+        const blob = new Blob([quizJson], { type: 'application/json' });
+        if (window.navigator.msSaveOrOpenBlob) { // IE & Edge
```

Comment:
please comment these lines in more details. Would it make sense to extract this into its own function?